src/OrbitSshQt/TunnelTest.cpp

```diff
@@ -48,7 +49,7 @@ class SshTunnelTest : public orbit_ssh_qt_test_utils::SshTestFixture {
 
     // socat with these options implements an echo server. It listen on incoming connections at a
     // random TCP port and replies to all incoming messages just by returning the same bytes.
-    task_.emplace(GetSession(), "socat -dd tcp-listen:0 exec:'/bin/cat'");
+    task_.emplace(GetSession(), "socat -dd tcp-listen:0,fork exec:'/bin/cat'");
```

Comment:
Without this, killing the echo server below might succeed or fail depending on whether `socat` has shut itself down already or not. To avoid this race condition, we tell socat to handle each incoming connection in a separate sub process and hence the originating socat process will still be around when the `kill` command gets issued and it will succeed - no race condition.